### DISCUSSION TITLE
There's a lack of scroll restoration in a README view.

### DISCUSSION THREAD

**Discussion by wanaiqing** [community/community#77268] [Mobile]
<details>
<summary>ℹ️ Additional Debug Info</summary>
GitHub for iOS, Version 1.140.0 (116339321)
Device: iPhone 12 mini (iOS 17.1.1)
</details>

It's bothering if you're reading a long README and you have to switch between apps.

**Comment by wanaiqing** [comment]
And it's weird that you can hack it by switching to another tab(any other tab than "home") before you go to another app.

**Reply by MarkPerez05** [reply]
😂! Been wanting to learn that
